app/assets/javascripts/search.js

```diff
@@ -46,7 +46,9 @@ function initialize(locale) {
 			// if(datum._type == "scientific_name")
 			// window.location.href = Routes.page_path(datum.id);
 			if (datum._type == "page")
-				window.location.href = locale + Routes.page_path(datum.id);
+				window.location.href = "/"+locale + "/pages/"+datum.id;
```

Comment:
"_type", as reported by Mirona, is not supported in the latest version of elastic-search. 
You need to consider.